framework/Dockerfile

```diff
@@ -1,7 +1,7 @@
 ARG BUILD_VERSION
 
-FROM idrinth/api-bench-build:${BUILD_VERSION} AS build
 FROM node:21.7.3-alpine AS node
```

Comment:
this needs to be written AFTER building, otherwise the copy makes no sense

Reply:
moved instruction after build